tensorboard/webapp/metrics/views/card_renderer/histogram_card_container.ts

```diff
@@ -107,7 +110,9 @@ export class HistogramCardContainer implements CardRenderer, OnInit {
   data$?: Observable<HistogramDatum[]>;
   mode$ = this.store.select(getMetricsHistogramMode);
   xAxisType$ = this.store.select(getMetricsXAxisType);
-  showFullSize = false;
+  readonly showFullSize$ = this.store
+    .select(getCardStateMap)
+    .pipe(map((map) => map[this.cardId]?.tableExpanded));
```

Comment:
`showFullSize` is not the same as `tableExpanded` these should be two different values.